components/avatar/index.tsx

```diff
@@ -102,11 +130,22 @@ export default class Avatar extends React.Component<AvatarProps, AvatarState> {
     }
   };
 
+  get size(): AvatarProps['size'] {
```

Comment:
We do not use `set` | `get` in antd component. Let's just use `getSize` as function instead.